testproject/Assets/Tests/Manual/testproject.manualtests.asmdef

```diff
@@ -1,8 +1,19 @@
 {
     "name": "TestProject.ManualTests",
+    "rootNamespace": "",
     "references": [
         "TestProject",
         "Unity.Multiplayer.MLAPI.Runtime",
-        "Unity.Multiplayer.MLAPI.Editor"
-    ]
+        "Unity.Multiplayer.MLAPI.Editor",
+        "Unity.Multiplayer.MLAPI.Prototyping"
+    ],
```

Comment:
I tend to suggest keeping asmdefs as simple and clear as possible.

In this case, it could simply be just:

```diff
{
    "name": "TestProject.ManualTests",
    "references": [
        "TestProject",
        "Unity.Multiplayer.MLAPI.Runtime",
        "Unity.Multiplayer.MLAPI.Editor",
+       "Unity.Multiplayer.MLAPI.Prototyping"
    ]
}
```